src/main/scala/mesosphere/marathon/core/plugin/PluginDescriptor.scala

```diff
@@ -0,0 +1,12 @@
+package mesosphere.marathon.core.plugin
```

Comment:
Nit: The file is called `PluginDescriptor.scala`, but there's no `PluginDescriptor` class anymore.